snapcraft/cli/lifecycle.py

```diff
@@ -188,5 +189,31 @@ def cleanbuild(remote, debug, **kwargs):
     lifecycle.cleanbuild(project_options, remote)
 
 
+@lifecyclecli.command()
+@click.option('--debug', is_flag=True,
+              help='Shells into the environment if the build fails.')
+def refresh(debug, **kwargs):
```

Comment:
This isn't part of the lifecycle. In order to prevent this module from becoming a catchall, I suggest you create a new module.